cognite/neat/rules/models/dms/_exporter.py

```diff
@@ -73,7 +73,7 @@ def to_schema(self) -> DMSSchema:
 
         output = DMSSchema(
             spaces=spaces,
-            data_models=dm.DataModelApplyList([data_model]),
+            data_model=data_model,
```

Comment:
this if not mistaken reverts back to the original idea of only having single data model